zbsprimarystorage/ansible/zbsp.py

```
@@ -0,0 +1,184 @@
+
+# name: install dependencies
+install_rpm_list = ""
+qemu_installed = yum_check_package("qemu-kvm", host_post_info)
+if not qemu_installed:
+    install_rpm_list += " %s" % qemu_alias.get(releasever, 'qemu-kvm')
+
+if zstack_repo != 'false':
+    command = """pkg_list=`rpm -q {} | grep "not installed" | awk '{{ print $2 }}'` && for pkg"""\
+            """ in $pkg_list; do yum --disablerepo=* --enablerepo={} install -y $pkg; done;"""\
+            .format(install_rpm_list, zstack_repo)
+    run_remote_command(command, host_post_info)
```

Ruff (0.8.2)

`run_remote_command` may be undefined, or defined from star imports

(F405)